typescript/sdk/src/core/message.ts

```diff
@@ -104,7 +98,7 @@ export class AbacusMessage {
   /**
    * The receipt of the TX that dispatched this message
    */
-  get receipt(): TransactionReceipt {
+  get receipt(): providers.TransactionReceipt {
```

Comment:
Optional: I feel like i would either prefer `ethers.providers.TransactionReceipt` or `TransactionReceipt` but not a mix of both